packages/x-date-pickers/src/internals/demo/DemoContainer.tsx

```diff
@@ -24,6 +25,10 @@ const getChildTypeFromChildName = (childName: string): PickersGridChildComponent
     return 'multi-panel-UI-view';
   }
 
+  if (childName.match(/^([A-Za-z]*)(DigitalClock)$/)) {
+    return 'Tall-UI-view';
```

Comment:
Do you have a better name for it? 🙈

Reply:
The `Ui-view` does layout the components in a flex `column`, while I wanted these to be in a `row`. 🤔